### PR TITLE
Send form

### PR DESCRIPTION
**Pull request by polyccon**
#6 You can now send the data to your favourite clinician, and if some sections have not been completed, the clinician will be informed for that as well.

### CODE REVIEW

File: src/controllers/send.js [hunk 1]
```diff
@@ -1,6 +1,11 @@
 exports.get = (req, res) => {
+  console.log(res.payload);
```
Comment (rebecacalvoquintero) [inline src/controllers/send.js]: ❤️  console.log

File: tests/send_email.test.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+test('checkEmail tests', (t) => {
+  const input = 'jen@gmail.com';
+  checkEmail(input);
+  t.pass('it should not throw an error for a valid email address');
+  t.end();
+});
+
+// testing the finish endpoint
+
+// test('check the email is sent', (t) => {
```
Comment (rebecacalvoquintero) [inline tests/send_email.test.js]: commented code

File: src/model/form_queries.js [hunk 3]
```diff
@@ -5,6 +5,21 @@ const getSection = (userId, sectionName) => {
   return dbConnection.oneOrNone(getGenerator(sectionName), [userId]);
 };
 
+const getForm = (userId) => {
+  const sections = ['about_me', 'symptoms', 'backgrounds', 'appointments', 'closing'];
+  const promises = sections.map((sectionName) => {
+    return getSection(userId, sectionName)
+          .then( (result) => {
```
Comment (dangerdak) [inline src/model/form_queries.js]: is this 'then' needed? :unicorn:

**Comment by Jen-Harris** [comment]
Now compatible with mobile image

**Pull request merged**